test-configs.yaml

```diff
@@ -41,15 +41,15 @@ file_systems:
 
   debian_buster_ramdisk:
     type: debian
-    ramdisk: 'buster/20190809.0/{arch}/rootfs.cpio.gz'
+    ramdisk: 'buster/20190823.0/{arch}/rootfs.cpio.gz'
```

Comment:
BTW This looks like a bug in Github as the rootfs URLs have already been updated on the master branch...